packages/auto-id/package.json

```diff
@@ -10,7 +10,10 @@
   },
   "dependencies": {
     "@autonomys/auto-utils": "workspace:*",
-    "@types/node": "^20.12.12"
+    "@peculiar/asn1-schema": "^2.3.8",
+    "@peculiar/asn1-x509": "^2.3.8",
+    "@types/node": "^20.12.12",
+    "asn1-ts": "^8.0.2"
```

Comment:
Is there only this package that does what we need?
It had a very low amount of download, and does not seem to be actively maintain...

Reply:
Yes I have matched with the requirement of `auto-kit` in python.

> It had a very low amount of download, and does not seem to be actively maintain...

We are using the X509 certificate from there. Didn't find any library that's regularly updated. Also, these algorithms are generally not updated frequently. They tend to be more stable. Hence, less/no update for long time.

Reply:
Yeah, it's not so much the low amount of recent commit that annoy me, more to see multiple Dependabot PR from many months ago, not merged and the low amount of downloads tbh.

Reply:
At this moment, we don't have any alternative that I can find. If you are able to find a better lib, please do share the package.

Reply:
What about https://www.npmjs.com/package/asn1js is it missing stuff?

Reply:
I went for a TS variant.
- https://www.npmjs.com/package/asn1js: last commit 2 years ago.
- https://www.npmjs.com/package/asn1-ts: 9 months ago.

Reply:
Regardless of it's name asn1js is fully type...
<img width="158" alt="image" src="https://github.com/subspace/auto-sdk/assets/82244926/41b5d284-68a6-4065-9540-a5ee0a087b74">
The repo has much more recent commit, they simply did not roll out recent version, we are still comparing a package with 1k or less download a week vs a package with 3.2M download a week...